Style/Frontend/index.md

```diff
@@ -0,0 +1,1451 @@
+  ```
+
+- Use template literals over string concatenation
+
+  ```jsx
+  // awful
+  const position = ['Your $AAPL position is valued at', value, '.'].join();
+
+  // bad
+  const position = 'Your $AAPL position is valued at ' + value + '.';
+
+  // good
+  const position = `Your $AAPL position is valued at ${value}.`;
+  ```
+
+- Don't unnecessarily escape strings
+
+  ```jsx
+  // bad
+  const str = '\'this\' \i\s \"quoted\"';
+  
+  // good
+  const str = '\'this\' is "quoted"';
+  ```
+
+## Typecasting
+
+- Typecase to number with `Number`, avoid `parseInt`
+
+  ```jsx
+  parseInt("23") // 23
+  parseInt("023") // 19 (octal)
+  parseInt("023", 10) // 23
+
+  Number('23') // 23
+  Number('023') // 23
+  ```
+
+- Typecast to string with `String`
+
+  ```jsx
+  const stock = {
+    price: 9,
+  };
+
+  // bad
+  const priceText = new String(stock.price); // typeof priceText is "object" not "string"
+  const priceText = stock.price + ''; // invokes this.reviewScore.valueOf()
+  const priceText = stock.price.toString(); // isn’t guaranteed to return a string
+
+  // good
+  const totalScore = String(stock.price);
+  ```
+
+- Typecast to bool with `!!`
+
+  > What this does is the first ! logically negates what was given which coerces the value to a bool, then it is negated again to get the original value, but how it would be if it were just a bool
+
+  ```jsx
+  const unsafe = null;
+  
+  // bad
+  const isSafe = new Boolean(unsafe);
+  const isSafe = Boolean(unsafe);
+  
+  // good
+  const isSafe = !!unsafe;
+  ```
+
+## Comparison Operators
+
+- Use `===` and `!==` over `==` and `!=` when not comparing to `null`
+
+  ```jsx
+  // bad
+  '1' == 1; // true
+
+  // good
+  '1' === 1; // false
+
+  // acceptable
+  1 != null;
+  ```
+
+- Conditional statements such as the `if` statement evaluate their expression using coercion with the `ToBoolean` abstract method and always follow these simple rules:
+
+  - **Objects** evaluate to **true**
+  - **Undefined** evaluates to **false**
+  - **Null** evaluates to **false**
+  - **Booleans** evaluate to **the value of the boolean**
+  - **Numbers** evaluate to **false** if **+0, -0, or NaN**, otherwise **true**
+  - **Strings** evaluate to **false** if an empty string `''`, otherwise **true**
+  - Read more from this [guide on equality in js](https://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/#more-2108) by Angus Croll
+
+- Use shortcuts for comparisons where it makes sense
+
+  ```jsx
+  // -- bool --
+  // bad
+  if (isValid === true) { ... }
+  
+  // good
+  if (isValid) { ... }
+  
+  // -- string --
+  // acceptable
+  if (name !== '') { ... }
+  
+  // good
+  if (name) { ... }
+  
+  // -- number --
+  // acceptable
+  if (collection.length > 0) { ... }
+  
+  // good
+  if (collection.length) { ... }
```

Comment:
I'd add an important note that the methods below are needed if you are writing a conditional in jsx. Otherwise you will get the "The text node needs to be wrapped in `<Text>`" error.

Reply:
Nice I actually felt it was deserving of it's own part so I added a conditionally rendering things section under the control statements section and a little reminder in this section that this code is not intended to be written in the JSX